relays/ethereum/src/sync.rs

```diff
@@ -228,9 +215,20 @@ mod tests {
 		H256::from_low_u64_le(1000 + number)
 	}
 
+	fn default_sync_params() -> HeadersSyncParams {
+		HeadersSyncParams {
+			max_future_headers_to_download: 128,
+			max_headers_in_submitted_status: 128,
+			max_headers_in_single_submit: 32,
+			max_headers_size_in_single_submit: 131_072,
```

Comment:
Where did this number come from?

Reply:
This is inside test code && originally comes from Ethereum -> Substrate sync. It is 32 * 4096 (`max_headers * max_header_and_receipts_size_we_expect`). For ethereum it gives ~6_500_000 gas / transaction (without execution cost). IMO should be ok too, given current limits.

Also just noticed that there values in configurations were wrong (sub2eth were used in eth2sub and vice versa) - fixed.

Reply:
Same thing I mentioned earlier with the other max header const

Reply:
On a second thought, in sub2eth, max_**_in_single_submit aren't that important now - there's single ethereum transaction per single substrate header. And these constants only make sense when several headers are submitted in single tx (like in eth2sub signed transactions). I've made conts with appropriate comments in `ethereum_sync_loop.rs`